core/db/curibio/alembic/versions/2ccc9fe21ec1_adding_plan_end_date_to_usage_.py

```diff
@@ -0,0 +1,30 @@
+"""Adding plan end date to usage_restrictions json object structure
+
+Revision ID: 2ccc9fe21ec1
+Revises: 1f23e047d2b0
+Create Date: 2023-01-25 14:40:08.612570
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "2ccc9fe21ec1"
```

Comment:
the database is at revision `89e819e0ecc2`, I would upgrade the database through alembic to make sure this works as expected. I think the dict needs another set of quotes to be in json

Reply:
Yeah, we want to run migrations we add before posting a PR. There may be some exceptions of course, but something small like this we want to make sure works. It also helps other people working in the repo if you run your migration script right away so that when they make a new one using `alembic revision -m "<msg>"` it uses the correct `down_revision`